### PR TITLE
update print_to_console to log to a text file

### PR DESCRIPTION
**Pull request by richbeales**
Uses python logging module to output progress to a text file log.txt

### Change Safety

- [ ] I have added tests to cover my changes
- [X] I have considered potential risks and mitigations for my changes

### CODE REVIEW

File: scripts/main.py [hunk 1]
```diff
@@ -355,7 +366,7 @@ def parse_arguments():
             f"COMMAND = {Fore.CYAN}{command_name}{Style.RESET_ALL}  ARGUMENTS = {Fore.CYAN}{arguments}{Style.RESET_ALL}")
 
     # Execute command
-    if command_name.lower() == "error":
```
Comment (nponeccop) [inline scripts/main.py]: This is an unrelated change

File: scripts/main.py [hunk 2]
```diff
@@ -381,3 +392,4 @@ def parse_arguments():
                 "system", "Unable to execute command"))
         print_to_console("SYSTEM: ", Fore.YELLOW, "Unable to execute command")
 
+
```
Comment (nponeccop) [inline scripts/main.py]: This is unrelated, too

**Review by nponeccop — CHANGES_REQUESTED** [review]
Remove the unrelated changes

File: scripts/main.py (hunk 1 above)
Comment (richbeales) [reply scripts/main.py]: done, had to pull latest into my branch.

File: scripts/main.py (hunk 2 above)
Comment (nponeccop) [reply scripts/main.py]: The added empty line is still there

**Review by nponeccop — CHANGES_REQUESTED** [review]

**Comment by nponeccop** [comment]
Does it just remove 2 empty lines?

**Review by nponeccop — APPROVED** [review]

**Comment by richbeales** [comment]
PR updated with latest main.py to resolve merge conflict.

**Review by nponeccop — APPROVED** [review]

**Review by Torantulino — APPROVED** [review]

**Comment by Torantulino** [comment]
Excellent, thank you @richbeales.
Cheers @nponeccop for your reviews and getting this focued!
